index.js

```diff
@@ -197,4 +197,36 @@ function popClose() {
   }
 }
 
-console.log(`${proDetail} ${popClose}`);
\ No newline at end of file
+console.log(`${proDetail} ${popClose}`);
+
+// form validation and localStorage
+
+const formEmail = document.querySelector('#email');
+const fullName = document.querySelector('#full-name');
+const Message = document.querySelector('#message');
+const form = document.querySelector('.contact-inputs');
+const errorMsg = document.querySelector('.error-message');
+
+const userData = window.localStorage.getItem('userData');
+
+const userDataFromLocal = JSON.parse(userData);
+fullName.value = userDataFromLocal.name;
+formEmail.value = userDataFromLocal.email;
+Message.value = userDataFromLocal.message;
+
+form.addEventListener('change', () => {
+  const dataObj = {
+    name: fullName.value,
+    email: formEmail.value,
+    message: Message.value,
+  };
+
```

Comment:
- [ ] While you did make efforts to fix this, the data is still not saved on local storage, and on reload, it disappears. 
- Instructions 
![image](https://user-images.githubusercontent.com/50990925/216734061-5a65c4d2-191d-4611-be06-12bf1517ddc6.png)
 - Please checkout this awesome [Resource ](https://dev.to/michaelburrows/how-to-save-data-in-localstorage-using-javascript-994) that has a step-by-step guide on how to fix this. 
 - Use your inspect on chrome developer tools to confirm just as I did in the view below :
 
 # Storage view :
 
![image](https://user-images.githubusercontent.com/50990925/216734280-8a0ebfb4-1137-4993-9f57-1b3f50b5fd40.png)

Reply:
Hello @Shedrack-Sunday, Thank you for the feedback.
After I changed the order of getItem and setItem and changed onchange event listener to keyup event listener, It's working and saving to localstorage as I type to the input fields 
![Screenshot 2023-02-04 091246](https://user-images.githubusercontent.com/77828236/216752451-abad321f-7d04-4236-bd70-9850429f2b01.png)

Thank you

Reply:
You're welcome